templates/emails/collective.virtualcard.assigned.hbs

```diff
@@ -0,0 +1,31 @@
+Subject: Virtual Card assigned to {{collective.name}}
+
+
+{{> header}}
+
+<center>
+  <h1>
+    {{host.name}} assigned a new Virtual Card to {{collective.name}}
+  </h1>
+</center>
+
+<p>
+  {{host.name}} assigned a new Virtual Card to your collective {{collective.name}}. This means that you also agreed with
```

Comment:
{{host.name}} assigned a new Virtual Card to your collective {{collective.name}}. As a reminder, you have agreed to their virtual card use policy and you can find it here {{LINK}}